apps/frontend/src/app/components/player-card/player-card.component.ts

```diff
@@ -16,7 +16,7 @@ import {
   styleUrl: './player-card.component.css'
 })
 export class PlayerCardComponent implements OnInit {
-  protected user: FullUser;
+  protected user?: FullUser;
```

Comment:
This is derived from I luservice's localUserSubject, which is a BS of `FullUser | null`. However, we know that this component should never be shown unless we have a LIU, due to the `@if {}`s in the template. If we don't, Angular should destroy the component. 

I think the correct approach is to keep the non-null type `user: FullUser`, and early return from the below `.subscribe` block if `user` is null there. I was considering throwing an error, since this component should be destroyed at roughly the same time, but I'm not sure exactly what order.